swift/ITSClient/Sources/ITSMobility/Mobility.swift

```diff
@@ -135,7 +135,7 @@ public actor Mobility {
                                                       referenceTime: now,
                                                       eventPosition: position,
                                                       stationType: mobilityConfiguration.stationType)
-        let situationContainer = SituationContainer(eventType: Cause(cause: cause))
+        let situationContainer = SituationContainer(eventType: cause)
         let denmMessage = DENMMessage(stationID: mobilityConfiguration.stationID,
```

Comment:
_managementContainer_ and _situationContainer_ of lines 140 and 141 must be shifted on space right as they are not aligned vertically with _stationID_